guides/common/attributes-titles.adoc

```diff
@@ -21,7 +21,7 @@
 :MonitoringDocTitle: Monitoring {Project} performance
 :PlanningDocTitle: Planning for {ProjectName}
 :ProvisioningDocTitle: Provisioning hosts
-:QuickstartDocTitle: Quickstart guide for {Project} on {install-on-os}
+:QuickstartDocTitle: Quick start guide for {Project} on {install-on-os}
```

Comment:
```suggestion
:QuickStartDocTitle: Quick start guide for {Project} on {install-on-os}
```

To make it consistent how we go from title to attribute name, we should capitalize the `s`.